fuzzers/dynamic_analysis/corpus/01

```diff
@@ -0,0 +1,6 @@
+GET /index.html HTTP/1.1
```

Comment:
Adding corpus to this guy is fine for me.
But can you use this?https://github.com/google/fuzzbench/blob/master/benchmarks/lcms_cms_transform_fuzzer/seeds/seed

Instead of an http request

Reply:
Thanks for the file! I will make a seperate PR to address this!